src/read.c

```diff
@@ -1285,134 +1323,116 @@ static avifBool avifDecoderGenerateImageGridTiles(avifDecoder * decoder, avifIma
     return AVIF_TRUE;
 }
 
-// Checks the grid consistency and copies the pixels from the tiles to the
-// dstImage. Only the freshly decoded tiles are considered, skipping the already
-// copied or not-yet-decoded tiles.
-static avifBool avifDecoderDataFillImageGrid(avifDecoderData * data,
-                                             avifImage * dstImage,
-                                             const avifTileInfo * info,
-                                             unsigned int oldDecodedTileCount,
-                                             avifBool alpha)
+// Allocates the dstImage based on the grid image requirements. Also verifies some spec compliance rules for grids.
+static avifResult avifDecoderDataAllocateGridImage(avifDecoderData * data, const avifTileInfo * info, avifImage * dstImage)
```

Comment:
Nit/optional: Add "Planes" to the end of the function name

Reply:
Done.